sys-libs/llvm-libunwind/metadata.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>zhanglei.april@gmail.com</email>
+		<name>Lei Zhang</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>toolchain@gentoo.org</email>
```

Comment:
Did you get permission from anyone in toolchain to add this? I don't think they care the slightest bit about anything but their GNU stuff. Add proxy-maint instead, since you need someone to push stuff for ya.